diff --git a/node_modules/@kalashshah/restapi/src/lib/chat/helpers/payloadHelper.ts b/node_modules/@kalashshah/restapi/src/lib/chat/helpers/payloadHelper.ts
index b322a8e..e3ed682 100644
--- a/node_modules/@kalashshah/restapi/src/lib/chat/helpers/payloadHelper.ts
+++ b/node_modules/@kalashshah/restapi/src/lib/chat/helpers/payloadHelper.ts
@@ -377,8 +377,8 @@ export const getAdminsList = (
       )
     : [];
 
-  const adminList = [...adminsFromMembers, ...adminsFromPendingMembers];
-  return adminList;
+  // const adminList = [...adminsFromMembers, ...adminsFromPendingMembers];
+  return [];
 };
 
 export const getSpaceAdminsList = (
@@ -405,14 +405,15 @@ export const getSpaceAdminsList = (
       )
     : [];
 
-  const adminList = [...adminsFromMembers, ...adminsFromPendingMembers];
-  return adminList;
+  // const adminList = [...adminsFromMembers, ...adminsFromPendingMembers];
+  return [];
 };
 
 export const convertToWalletAddressList = (
   memberList: { wallet: string }[]
 ): string[] => {
-  return memberList ? memberList.map((member) => member.wallet) : [];
+  // return memberList ? memberList.map((member) => member.wallet) : [];
+  return [];
 };
 
 export const getMembersList = (
@@ -430,7 +431,8 @@ export const getMembersList = (
   }[]
 ): Array<string> => {
   const allMembers = [...(members || []), ...(pendingMembers || [])];
-  return convertToWalletAddressList(allMembers);
+  // return convertToWalletAddressList(allMembers);
+  return [];
 };
 
 export const getSpacesMembersList = (
@@ -448,5 +450,6 @@ export const getSpacesMembersList = (
   }[]
 ): Array<string> => {
   const allMembers = [...(members || []), ...(pendingMembers || [])];
-  return convertToWalletAddressList(allMembers);
+  // return convertToWalletAddressList(allMembers);
+  return [];
 };
